pybennu fails to install without libzmq5-dev

I'm not sure what has changed, but the bennu image is now failing to build due to the pybennu deb package failing to install. The missing dependency it needs is `libzmq5-dev`. It makes sense to me why this is needed but I'm not sure why it has been working up to this point without it.

I think we just need to add `libzmq5-dev` to the dependencies [here](https://github.com/sandialabs/sceptre-bennu/blob/938057596b0ecdf2278590bc8ee3f2833476971c/cmake/CpackConfig.cmake#L17)

@GhostofGoes 
I can do that PR but just wanted to check if you can confirm this or if there is any other history here you know of.

More details on the failure below, I've tested on multiple systems so I'm pretty confident. I'll open a PR soon.

```
2025-02-19T00:57:47.4356877Z DEBUG STDERR: b'ile__=\'"\'"\'/tmp/pip-req-build-rjcv77k6/setup.py\'"\'"\';f=getattr(tokenize, \'"\'"\'open\'"\'"\', open)(__file__);code=f.read().replace(\'"\'"\'\\r\\n\'"\'"\', \'"\'"\'\\n\'"\'"\');f.close();exec(compile(code, __file__, \'"\'"\'exec\'"\'"\'))\' install --record /tmp/pip-record-siwwq76z/install-record.txt --single-version-externally-managed --compile --install-headers /usr/local/include/python3.8/helics Check the logs for full command output.\ndpkg: error processing package pybennu (--configure):\n installed pybennu package post-installation script subprocess returned error exit status 1\n'
```

```
2025-02-19T00:57:47.2745287Z DEBUG STDERR: b'st-1.71.0/BoostConfig.cmake (found suitable version "1.71.0", minimum required is "1.58")\n    -- Looking for pthread.h\n    -- Looking for pthread.h - found\n    -- Performing Test CMAKE_HAVE_LIBC_PTHREAD\n    -- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed\n    -- Check if compiler accepts -pthread\n    -- Check if compiler accepts -pthread - yes\n    -- Found Threads: TRUE\n    CMake Error at CMakeLists.txt:540 (message):\n      ZeroMQ not found, needed to enable the ZMQ Core\n    \n    \n    -- Configuring incomplete, errors occurred!\n    See also "/tmp/pip-req-build-rjcv77k6/build/temp.linux-x86_64-3.8/CMakeFiles/CMakeOutput.log".\n    See also "/tmp/pip-req-build-rjcv77k6/build/temp.linux-x86_64-3.8/CMakeFiles/CMakeError.log".\n    Traceback (most recent call last):\n      File "<string>", line 1, in <module>\n      File "/tmp/pip-req-build-rjcv77k6/setup.py", line 287, in <module>\n        setup(\n      File "/usr/lib/python3/dist-packages/setuptools/__init__.py", line 144, in setup\n        return distutils.core.set'
```

--- a/src/pybennu/Makefile
+++ b/src/pybennu/Makefile
@@ -122,6 +122,7 @@ deb: packagetools
 		-d python3-setuptools \
 		-d wget \
 		-d libboost-python-dev \
+		-d libzmq5-dev \
 		-p '$(subst __colon__,:,$(DIST_DIR)/$(PACKAGE_FILENAME))' \
 		--name '$(PACKAGE_NAME)' \
 		$(call iter,$(PACKAGE_VENDOR),--vendor) \

